specifications/active/aRMT-1.0.0.yml

```diff
@@ -6,6 +6,46 @@ version: 1.0.0
 assessment_type: QUESTIONNAIRE
 doc: aRMT Questionnaires definition. Includes Personal Health Questionnaire Depression Scale (PHQ-8), Experience sampling method (ESM) and RSES.
 data:
+  - type: THINC-IT
+    topic: notification_thinc_it
+    value_schema: .active.notification.Notification
+    questions:
+      - lead: Time for Thinc-it
+        content: It's time to complete the Thinc-it app.
+        widget: INFO
+  - type: ROMBERG_TEST
+    topic: task_romberg_test
+    value_schema: .active.task.Task
+    questions:
+      - lead: Romberg's Test
+        content: It's time to complete the Thinc-it app.
+        widget: INFO
+      - lead: Romberg's Test
+        content: Keep your eyes open.
+        widget: TIMED
+      - lead: Romberg's Test
+        content: Close your eyes.
+        widget: TIMED
+  - type: 2MW_TEST
+    topic: task_2MW_test
+    value_schema: .active.task.Task
+    questions:
+      - lead: 2MW Test
+        content: Null
+        widget: INFO
+      - lead: 2MW Test
+        content: Walk back and forth.
+        widget: TIMED
+  - type: TANDEM_WALKING_TEST
+    topic: task_tandem_walking_test
+    value_schema: .active.task.Task
+    questions:
```

Comment:
What is the use-case of adding the definitions of questions in the specification. I thought these were in the https://github.com/RADAR-base/RADAR-REDCap-aRMT-Definitions . Whats the point of keeping these in 2 places. Also these will have to be updated manually and may go out of sync with the other.

Reply:
I was wondering the same thing earlier; it would reduce a quite some code from this repo if questionnaires definitions are stored elsewhere. Perhaps replacing we can just add the name or URL of the questionnaire as defined in the REDCAP definitions repo?

Reply:
Sounds good. I will create a new PR for these changes